SPECS/nexus3-oss.spec

```diff
@@ -169,8 +169,35 @@ rm -rf $RPM_BUILD_ROOT
 %endif
 
 %changelog
-* Fri Feb 10 2023 Julio González Gil <packages@juliogonzalez.es> - 3.47.0.01-1
-- Update to Nexus 3.47.0-01
+* Tue Feb 28 2023 Julio González Gil <packages@juliogonzalez.es> - 3.48.0.01-1
+- Update to Nexus 3.48.0-01
+- Bugfixes:
+  * NEXUS-36573: create a repository of any format triggers a complete Helm
+                 index rebuild
+  * NEXUS-36998: dotnet add command against nuget v2 group repo is very slow
+                 on newDB
+  * NEXUS-37617: loading welcome page can make frequent calls to database
+                 which may lead to ui latency in some circumstances
+  * NEXUS-37810: strict content validation may fail on exe
+                 application/x-sharedlib protobuf protoc
+- Improvements:
+  * NEXUS-37535: Removed Legacy SQL Maven Metadata Builder
+  * Improved NuGet v2 Performance on PostgreSQL (PRO Only)
+  * New and improved Content Replication (PRO Only)
+    While help documentation for legacy replication remains available,
+    it will be unavailable for general use, and will be eventually removed.
+    Reach out to your CSE if you are still using legacy replication and
+    would like to continue using it
+- Notes:
+  After upgrading to 3.48.0, those using an H2 or PostgreSQL database will
+  need to run a task for each existing Apt, Helm, and Yum repository in
+  order to rebuild their metadata:
+  * Apt: Rebuild Apt metadata for each Apt repository
+  * Helm: Rebuild Helm metadata for each Helm repository
+  * Repair: Rebuild Yum rebuild metadata (repodata) for each Yum repository
+
+* Fri Feb 10 2023 Julio González Gil <packages@juliogonzalez.es> - 3.47.1.01-1
```

Comment:
Seems there was a mistake with the version at the changelog, caused by a copy & paste.

Did not affect the user experience, as `Version` and `Source0` were right.